precompiles/xcm-utils/XcmUtils.sol

```diff
@@ -42,4 +42,16 @@ interface XcmUtils {
         external
         view
         returns (uint256 unitsPerSecond);
+
+    /// Execute custom xcm message
+    /// @custom:selector 34334a02
+    /// @param message The versioned message to be executed scale encoded
+    /// @param maxWeight The maximum weight to be consumed
+    function xcmExecute(bytes memory message, uint64 maxWeight) external;
```

Comment:
Could be worth a brief note that this can't be called from smart contract. I'm inclined to question why we should add it at all here, what good could it do? It won't help anyone call directly from EoA, and it will fail in any other case...